### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by ap221882**
Made a basic webpage rendering the info of TEAM-A members.

### CODE REVIEW

File: Day4/src/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (nitishnivedan) [inline Day4/src/App.js:1]: can leave  a empty line between library imports ( line 1) and local files import (line 2,3,4) for better readability

File: Day4/src/App.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import "./App.styles.css";
+import Header from "./components/Header/Header";
+import HomePage from "./pages/HomePage/HomePage";
+
+const App = () => {
+  return (
```
Comment (nitishnivedan) [inline Day4/src/App.js:8]: App component can be returned without the return key word. 
e.g.:  const App = ()=> (<div>
...
</div>);

Comment (ap221882) [reply Day4/src/App.js:8]: Done with this change

File: Day4/src/App.js (hunk 1 above)
Comment (ap221882) [reply Day4/src/App.js:1]: Done.

**Comment by ap221882** [comment]
Merged with adding the reviews suggested.

**Pull request merged**